translations/english/core-commands.ts

```diff
@@ -83,19 +83,14 @@ export const translations: Translations = {
 		"${user.name} hid the replay of this battle.": "",
 		"You can only do this in battle rooms.": "",
 		"You can only add a Player to unrated battles.": "",
-		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
```

Comment:
This translation seems to be still in use: https://github.com/smogon/pokemon-showdown/blob/0475b38ae8ad2cd1ea0b2849972a9c821bd6a1de/server/chat-commands/core.ts#L952